web/src/blueslip.ts

```diff
@@ -102,20 +102,20 @@ export function error(
     more_info?: object | undefined,
     original_error?: unknown | undefined,
 ): void {
+    // Log the Sentry error before the console warning, so we don't
+    // end up with a doubled message in the Sentry logs.
+    Sentry.setContext("more_info", more_info === undefined ? null : more_info);
+
     // original_error could be of any type, because you can "raise"
     // any type -- something we do see in practice with the error
     // object being "dead": https://github.com/zulip/zulip/issues/18374
-    let exception: string | Error = msg;
+    // We only mark it as the "cause" if it's an Error.
     if (original_error !== undefined && original_error instanceof Error) {
```

Comment:
This check isn’t necessary. An `Error` has a stack trace even if its `cause` does not.